pkg/state/state.go

```diff
@@ -24,6 +24,10 @@ const (
 	Destroyed
 )
 
+func (eventType EventType) String() string {
+	return [...]string{"Created", "Updated", "Destroyed"}[eventType]
```

Comment:
Why not use `stringer` tool? https://pkg.go.dev/golang.org/x/tools/cmd/stringer

Reply:
never used, that, good point, if this requires changing build steps, might be too much work for this PR, but certainly something for the future

Reply:
https://github.com/talos-systems/kres/issues/53